content/docs/testing-farm.md

```diff
@@ -108,6 +118,7 @@ You can verify that the option is processed correctly using a [YAML parser](http
   when you have multiple `tests` jobs with different configuration. For
   example if you set this to `e2e-tests`, then a check run for Rawhide would be
   named `testing-farm:fedora-rawhide-x86_64:e2e-tests`.
+* **manual_trigger** - Whether to trigger Testing Farm jobs only via pull request comment or not (default to false). Trigger phrase is `/packit test` (`/packit-stg test` for stage).
```

Comment:
How do you specify the job which should be triggered? Should we include an example for that as well?

Reply:
as @Frawless wrote, currently, all test jobs will be triggered. 

The follow-up issue that would cover the functionality of specifying the job: https://github.com/packit/packit-service/issues/1754#issuecomment-1522949476

Reply:
Ah, I see, so this is the first part. Thanks for clarification!

Reply:
Thanks for the heads-up. Very nice! Looking forward to trying this out :) Is it supposed to be enabled on the production instance today?

Reply:
The feature was merged last week so I think it should be in prod already.